hedera-node/hedera-addressbook-service-impl/src/test/java/com/hedera/node/app/service/addressbook/impl/test/handlers/AddressBookTestBase.java

```diff
@@ -184,4 +206,12 @@ protected Node createNode() {
                 .weight(0)
                 .build();
     }
+
+    static Key mockPayerLookup(Key key, AccountID accountId, ReadableAccountStore accountStore)
```

Comment:
This can be removed after you removed the code from preHandle() to handle(). Same to A_COMPLEX_KEY above.

Reply:
removed